### ISSUE TITLE
Add automatic scrolling for the code-input

### ISSUE THREAD

**Issue by ghost** [SStalker/EVOS#155]

**Comment by apollo7321** [comment]
Habt ihr ein Beispiel-Screenshot?

**Comment by lhannigbrinck** [comment]
Nein, aber es war heute ein "DAU-Dummy" hier und der hat erst ein paar zeielen geschrieben und dann versucht code einzufügen, und man hat halt nciht gesehen, das die tags einfach auserhalb des sichtfeldes eingefügt wurden.

**Comment by apollo7321** [comment]
Hmm, man könnte das eventuell noch umschreiben, sodass der markierte Code umschlossen wird. Dazu hilfreich: https://wiki.selfhtml.org/wiki/JavaScript/Anwendung_und_Praxis/Formulare:_Text_an_Cursorposition_einf%C3%BCgen
